build-linux/deb/postinst

```diff
@@ -54,9 +58,13 @@ if [ "$1" = "configure" ]; then
     # users cannot be seen, but set to -wx for enqueue-related directories
     # to allow any user to write events.
     chown -R pdagent:pdagent /var/lib/pdagent /var/log/pdagent
-    chmod 750 /var/lib/pdagent/outqueue/err /var/lib/pdagent/outqueue/suc
-    chmod 753 /var/lib/pdagent/outqueue/tmp /var/lib/pdagent/outqueue/pdq
-
+    chown -R pdagent:pdagent /var/lib/pdagent/outqueue/err /var/lib/pdagent/outqueue/suc
```

Comment:
Rethinking this, we might not want to change ownership here and open up access to these files instead.

This script is run as part of the post-install process on users' machines, so they might not have a `pdagent` user as we do in the docker environment, or might want to be able to run `pd-send` from non-`pdagent` users.

Reply:
I think that could work. So just remove this line perhaps and the chmod changes below should cover it? I wonder if we'll need to add this line to the dockerfile maybe

Reply:
Yeah, preferring chmodding things open rather than chowning should work.

Now that the docker image installs the package we probably won't need to since it'll pick up any permissions changes from this file.

Reply:
@dobs 
The deb/rpm will build the pdagent user, those directories have those permissions to protect the data in the directory in the case of a standard multi-user unix system (or VPC).

The pdagentd daemon does a setuid to the pdagent user as part of the systemd configuration.

So the original permissions I think were carefully selected.

Reply:
Sounds good then -- there might not be a case for having arbitrary users run these commands.